source/includes/harvest/_users.md

```diff
@@ -86,6 +86,72 @@ List all of an organization's Greenhouse users.
 <br>
 [See noteworthy response attributes.] (#the-user-object)
 
+## GET: List Job Permissions for User
+
+```shell
+curl 'https://harvest.greenhouse.io/v1/users/{id}/permissions/jobs'
+-H "Authorization: Basic MGQwMzFkODIyN2VhZmE2MWRjMzc1YTZjMmUwNjdlMjQ6"
+```
+
+```json
+[
+  {
+    "id": 7879576,
+    "job_id": 80722,
+    "user_role_id": 4730
+  },
+  {
+    "id": 7879577,
+    "job_id": 83475,
+    "user_role_id": 4730
+  },
+  {
+    "id": 7879579,
+    "job_id": 146048,
+    "user_role_id": 4730
+  },
+  {
+    "id": 7879580,
+    "job_id": 87904,
+    "user_role_id": 4730
+  },
+  {
+    "id": 7879582,
+    "job_id": 116958,
+    "user_role_id": 4730
+  },
+  {
+    "id": 7879583,
+    "job_id": 82318,
+    "user_role_id": 4730
+  }
+]
+```
+
+List the job permissions for a given user.
+
+### HTTP Request
+
+`GET https://harvest.greenhouse.io/v1/users/{id}/permissions/jobs`
+
+### URL Parameters
+
+Parameter | Description
+--------- | -----------
+id | The ID of the user whose job permissions to retrieve
+
+### HTTP Response
```

Comment:
We don't really have anything like this in our docs so let me know what you think.